NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 [Unreleased]: https://github.com/electron-userland/electron-packager/compare/v12.0.0...master
 
+* Added `asar.filename` to support pre-built asar files.
```

Comment:
Could you please adjust this to conform to the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format?